Suko/Cells/SUKMovieToAnimeCollectionViewCell.m

```diff
@@ -0,0 +1,23 @@
+//
+//  SUKMovieToAnimeCollectionViewCell.m
+//  Suko
+//
+//  Created by Alice Zhang on 7/22/22.
+//
+
+#import "SUKMovieToAnimeCollectionViewCell.h"
+#import "UIImageView+AFNetworking.h"
+
+@implementation SUKMovieToAnimeCollectionViewCell
+
```

Comment:
You probably want to set up an init and/or a layoutSubviews method in order initialize and layout all of the subviews.

Reply:
In this case, storyboard did that for me! But since I am using swiping cards now, I have actually already deleted this collection cell file!